edx-platform/config/cms/docker_run_development.py

```diff
@@ -0,0 +1,21 @@
+from docker_run_production import *
+
+DEBUG = True
```

Comment:
In fact, it should be false to prevent an obscure JS error with unprocessed statics (see openfun/openedx-docker/issues/105)

```suggestion
DEBUG = False
```

Reply:
This is dev settings, and should work as it. also DEBUG=True is not the real issue

Reply:
> DEBUG=True is not the real issue

True, but it's an openedx issue. Let's be pragmatic on this to test this view.